### ISSUE TITLE
BarGraph: Needs max/min Zoom options/or zoom slider.

### ISSUE THREAD

**Issue by cefiar** [janesma/apitrace#22]
While I know you can adjust the zoom using the - and + on either side of the scroll bar under bargraph, the ability to zoom all the way out or all the way in would be advantageous.

Perhaps a dedicated zoom scroll bar vertically on the side of the BarGraph? This way you could drag it top or bottom, click on a specific place to set the zoom, or use + and - to increment in steps (for fine control).

**Comment by janesma** [comment]
I agree that this is annoying.  My issue is that there is no "maximum zoom".  You can keep zooming in as much as you want.  I don't know how to set the values for a slider.

It might be better if the zoom factor accelerated if you continue to scroll.  It's frustrating that mousepad (2 fingered) scrolling is much faster than when you use a scroll wheel, making it hard to have a single policy.
